packages/atomic/storybookUtils/commerce-interface-wrapper.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  CommerceEngineConfiguration,
+  getOrganizationEndpoints,
+} from '@coveo/headless/commerce';
+import {within} from '@storybook/test';
+import {Decorator, StoryContext} from '@storybook/web-components';
+import {html} from 'lit/static-html.js';
+import type * as _ from '../src/components.d.ts';
+
+const getSampleCommerceEngineConfiguration =
+  (): CommerceEngineConfiguration => ({
+    accessToken: 'xxc481d5de-16cb-4290-bd78-45345319d94c',
+    organizationId: 'barcasportsmcy01fvu',
+    organizationEndpoints: getOrganizationEndpoints(
+      'barcasportsmcy01fvu',
+      'dev'
+    ),
+    analytics: {
+      trackingId: 'sports',
+    },
+    context: {
+      language: 'en',
+      country: 'US',
+      currency: 'USD',
+      view: {
+        url: 'https://github.com/coveo/ui-kit',
+        referrer: document.referrer,
+      },
+    },
+    cart: {
+      items: [
+        {
+          productId: 'SP01057_00001',
+          quantity: 1,
+          name: 'Kayaker Canoe',
+          price: 800,
+        },
+        {
+          productId: 'SP00081_00001',
+          quantity: 1,
+          name: 'Bamboo Canoe Paddle',
+          price: 120,
+        },
+        {
+          productId: 'SP04236_00005',
+          quantity: 1,
+          name: 'Eco-Brave Rashguard',
+          price: 33,
+        },
+        {
+          productId: 'SP04236_00005',
+          quantity: 1,
+          name: 'Eco-Brave Rashguard',
+          price: 33,
+        },
+      ],
+    },
+  });
```

Comment:
Could/should be improved by having a similar method in Coveo Headless (and thus an org that we are OK with customer touching)